recipe/bld.bat

```diff
@@ -1,4 +1,13 @@
-cmake -G "NMake Makefiles" -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR% -D BUILD_SHARED_LIBS=ON -D BUILD_TZ_LIB=ON -D USE_SYSTEM_TZ_DB=ON -D CMAKE_BUILD_TYPE=Release
+export CXXFLAGS="$CXXFLAGS -DINSTALL=$PREFIX\share\zoneinfo\"
+
+cmake -G "NMake Makefiles" ^
+     -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+     %SRC_DIR% ^
+     -D BUILD_SHARED_LIBS=ON ^
+     -D BUILD_TZ_LIB=ON ^
+     -D USE_SYSTEM_TZ_DB=ON ^
```

Comment:
Unrelated: `USE_SYSTEM_TZ_DB` is unsupported on Windows, it seems: https://github.com/HowardHinnant/date/pull/611

cc @pidefrem